src/sentry/static/sentry/app/views/projects/redirectLegacyProjectSavedSearchRoute.jsx

```diff
@@ -0,0 +1,190 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import _ from 'lodash';
+
+import {t} from 'app/locale';
+import withApi from 'app/utils/withApi';
+import LoadingError from 'app/components/loadingError';
+import {fetchProjectSavedSearches} from 'app/actionCreators/savedSearches';
+import {analytics} from 'app/utils/analytics';
+import Alert from 'app/components/alert';
+import Redirect from 'app/utils/redirect';
+
+import {ProjectDetails} from './redirectLegacyProjectRoute';
+
+const DEFAULT_SORT = 'date';
+const DEFAULT_STATS_PERIOD = '24h';
+const STATS_PERIODS = new Set(['14d', '24h']);
+
+const redirectLegacyProjectSavedSearchRoute = generateRedirectRoute => {
+  class RedirectLegacyProjectSavedSearchRoute extends React.Component {
```

Comment:
Doh I would double check with @saragilford 	on this but I think we don't need to worry about legacy saved searches. https://www.notion.so/sentry/Spec-Recent-and-Saved-Searches-af224779a81149259e93986586863cf5#354c2af0e5d1453f94218fd63caa9dd5

Reply:
For posterity, I'm removing the redirect for this.